apple/internal/experimental_mixed_language_library.bzl

```diff
@@ -275,6 +278,11 @@ target only contains Objective-C files.""")
         "-import-underlying-module",
     ]
 
+    # Modules is not enabled if a custom module map is present even with
```

Comment:
I noticed the same thing when I was looking into this a while ago. Can you file an issue in Bazel core just to track that this oddity exists?

Reply:
Yep will do!

Reply:
It seems to be explicitly checked against: https://github.com/bazelbuild/bazel/blob/18d01e7f6d8a3f5b4b4487e9d61a6d4d0f74f33a/src/main/java/com/google/devtools/build/lib/rules/objc/CompilationSupport.java#L1280 but its odd because without `-fmodules` you cant `@import` even if you have a module map

Reply:
I had figured it intentional given the user is making the call to use a custom module map. But 🤷‍♂️.

Reply:
https://github.com/bazelbuild/bazel/issues/20703

Reply:
Mind adding the new issue in the code comment before merging, please?